mithril-test-lab/mithril-devnet/VERSION

```diff
@@ -1 +1 @@
-0.3.9
+0.3.10
```

Comment:
I think that you should bump the minor version since those changes are breaking for nodes version below 9.

Reply:
Done in https://github.com/input-output-hk/mithril/pull/1912/commits/0a38da4024540d6b18732b54cd3b80675742be72